http/vertx/scripts/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
Is this script in any way different from the one we have for the regular Kafka clients? If not, should we have it just once?